src/QirTools/pyqir/README.md

```diff
@@ -15,68 +15,148 @@ The Rust project uses:
 
 This should allow us to load the QIR and replace the entrypoint method body with the circuit translation.
 
-## Building
+# Building
 
-This repo includes a dev container which will install all base requirements. If you with to build locally without a dev container, you'll need to install:
+## Dev Container
 
-- LLVM 11 ([Ubuntu](https://apt.llvm.org/), Windows)
-- Rust > 1.48
-- Python 3.6+
+This repo includes a dev container which will install all base requirements.
 
-## Dev workflow
+- [Visual Studio Code](https://code.visualstudio.com/)
+- Remote-Containers Extension (`ms-vscode-remote.remote-containers`)
+- Docker
 
-### Building and Running Tests
+Open `src\QirTools\pyqir` in VS Code and press `Ctrl + Shift + P` to open the command pallet and launch `Remote-Containers: Rebuild and Reopen in Container`. This will open a new VS Code window and build the development environment in a container.
 
-In order to run the python tests, the module needs to be installed. This setup will generate and install an egg. The `./build_wheels.sh` script can also be run if you wish to build and install a wheel.
+## Local Environment
 
-If this is the first time you've loaded the code we need to install the python dev packages:
+If you wish to set up your own local environment on Windows or Linux, follow one of the sections below.
+
+### Linux (Ubuntu)
+
+Install python and libs:
+
+```bash
+sudo apt-get install -y --no-install-recommends python3-dev python3-pip
+python3 -m pip install -U pip
+python3 -m pip install maturin tox
+```
+
+Install Rust from [rustup](https://rustup.rs/).
+
+Install LLVM via [LLVM's apt source](#llvm-apt) or via [conda/miniconda](#linux-miniconda):
+
+#### LLVM Apt
+
+```bash
+sudo apt-get install -y --no-install-recommends lsb-release wget software-properties-common
+wget https://apt.llvm.org/llvm.sh
+chmod +x llvm.sh
+sudo ./llvm.sh 11
+
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends cmake ninja-build clang-11 clang-tidy-11 build-essential
+```
+
+#### Linux Miniconda
+
+Install [miniconda](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links).
+
+In a command prompt:
+
+```bash
+conda install -y -c conda-forge llvm-tools=11.1.0 llvmdev=11.1.0 clang=11.1.0 cmake=3.20.4 ninja=1.10.2
+```
+
+### Windows
+
+Install Python 3.6+ from [Python.org](https://www.python.org/downloads/).
+
+In an Administrator command prompt:
 
 ```bash
-python3 -m pip install -r requirements-dev.txt
+python -m pip install maturin tox
 ```
 
-After which we can build everything and install the module locally:
+Install Rust from [rustup](https://rustup.rs/).
+
+Install [miniconda](https://docs.conda.io/en/latest/miniconda.html#latest-miniconda-installer-links).
+
+In a command prompt:
 
 ```bash
-python3 setup.py install --user
+conda install -y -c conda-forge llvm-tools=11.1.0 llvmdev=11.1.0 clang=11.1.0 cmake=3.20.4 ninja=1.10.2
```

Comment:
It may also be helpful to provide an environment.yml file with these versions locked.